Datadog/Example/Scenarios/CrashReporting/CrashReporting/CrashReportingViewController.swift

```diff
@@ -82,4 +84,22 @@ internal class CrashReportingViewController: UIViewController {
     @objc func dereferenceNullPointer() {
         objc.dereferenceNullPointer()
     }
+
+    @objc func infiniteRecursiveCall() {
+        _ = recursiveCall(parameter: 0)
+    }
+
+    private var heap: [String] = []
+
+    func recursiveCall(parameter: Int) -> Int {
+        if parameter > 1_000_000 {
+            heap.append("abcdefghijkl")
+            return recursiveCall(parameter: parameter - 1)
+        } else if parameter <= 1_000_000 {
+            heap.append("abcdefghijkl")
+            return recursiveCall(parameter: parameter + 1)
+        } else {
+            return 0 // not reachable, but mutes compiler warning
+        }
```

Comment:
trivial:
```suggestion
        } else {
            heap.append("abcdefghijkl")
            return recursiveCall(parameter: parameter + 1)
        }
```

Reply:
Indeed, my mind was too much into recursion and I haven't noticed that, thx!